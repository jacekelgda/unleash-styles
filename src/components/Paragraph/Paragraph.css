.Paragraph {
	font: 14px 'Source Sans Pro', sans-serif;
	color: #6e6d72;
	margin: 0;
}
@media screen and (max-width: 767px) {
	.Paragraph {
		margin-left: 83px;
	}
}
@media screen and (min-width: 768px) {
	.Paragraph {
		font-size: 18px;
		margin: 20px 0 23px;
	}
}
