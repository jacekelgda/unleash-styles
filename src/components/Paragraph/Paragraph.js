import React from 'react'
import PropTypes from 'prop-types'
import styles from './Paragraph.css'

/**
 * Paragraph
 */
function Paragraph (props) {
  return <p className={styles.Paragraph}>{props.text}</p>
}

Paragraph.propTypes = {
  /** text to display */
  text: PropTypes.string.isRequired
}

export default Paragraph
