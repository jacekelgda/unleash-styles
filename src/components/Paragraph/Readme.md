Paragraph

    <Paragraph text='Lorem Ipsum' />
