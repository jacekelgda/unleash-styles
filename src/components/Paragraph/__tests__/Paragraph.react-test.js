/* global test, expect */
import React from 'react'
import { shallow } from 'enzyme'

import Paragraph from '../Paragraph'

test('Paragraph renders correctly', () => {
  const wrapper = shallow(<Paragraph text='Lorem Ipsum' />)
  expect(wrapper).toMatchSnapshot()
})
