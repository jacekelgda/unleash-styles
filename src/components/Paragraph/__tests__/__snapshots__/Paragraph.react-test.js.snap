// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <p
    className="Paragraph"
>
    Lorem Ipsum
</p>,
  "nodes": Array [
    <p
      className="Paragraph"
>
      Lorem Ipsum
</p>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <Paragraph
        text="Lorem Ipsum"
/>,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {
          "text": "Lorem Ipsum",
        },
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <p
          className="Paragraph"
>
          Lorem Ipsum
</p>,
        "_debugID": 2,
        "_renderedOutput": <p
          className="Paragraph"
>
          Lorem Ipsum
</p>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Paragraph
    text="Lorem Ipsum"
/>,
}
`;
